### ISSUE TITLE
ALTER TABLE ADD [COLUMN] statement requires COLUMN keyword

### ISSUE THREAD

**Issue by olavloite** [GoogleCloudPlatform/cloud-spanner-emulator#16]
The following DDL statement is accepted by Cloud Spanner and rejected by the emulator:

```sql
ALTER TABLE Singers ADD FullName STRING(500)
```

The emulator returns the following error for the above statement: `INVALID_ARGUMENT: Error parsing Spanner DDL statement: ALTER TABLE Singers ADD FullName STRING(500) : Syntax error on line 1, column 25: Expecting 'CHECK' but found 'FullName' [Failed SQL: (3) ALTER TABLE Singers ADD FullName STRING(500)]`

The emulator requires the statement to include the `COLUMN` keyword, although it according to the [documentation](https://cloud.google.com/spanner/docs/data-definition-language#alter_table) should be optional.

The following statement is successful on both the emulator and Cloud Spanner:

```sql
ALTER TABLE Singers ADD COLUMN FullName STRING(500)
```

**Comment by Vizerai** [comment]
Thanks for pointing this out. We will fix this in the next release.

**Comment by tfindlay-au** [comment]
This issue seems to have been re-introduced where by the ADD clause requires the COLUMN key word.

The Spanner documentation still denotes this as optional:
https://cloud.google.com/spanner/docs/data-definition-language#table_statements

Further, this statement is valid and works in Cloud Spanner, but not in the emulator.
```
> ALTER TABLE MY_TABLE ADD MY_COLUMN INT64
[2021-06-08 18:13:54] [3] INVALID_ARGUMENT: io.grpc.StatusRuntimeException: INVALID_ARGUMENT: Error parsing Spanner DDL statement: ALTER TABLE MY_TABLE ADD MY_COLUMN INT64 : Syntax error on line 1, column 36: Expecting 'CHECK' but found 'MY_COLUMN'
```

I noticed the test coverage for this original issue was updated to include the COLUMN key word.

**Comment by jacobdufault-google** [comment]
The fix for this is included in the 1.3 release (738d9cc62038a84981f749acf9b43d86c72fbf12).
